pyproject.toml

```
@@ -28,8 +28,11 @@ lightning-bolts = "^0.3.4"
 palkit = "^0.3.8"
 torch = "*"
 torchvision = "*"
+torchaudio = "*"
 gdown = "^3.13.0"
 opencv-python = "^4.5.3"
+soundfile = {version = "*", markers = "platform_system == 'Windows'"}
+sox = {version = "*", markers = "platform_system == 'Linux' or platform_system == 'macOS'"}
```

I've not had to use this before with poetry, well done for working it out. 
```suggestion
soundfile = {version = "*", markers = "platform_system == 'Windows'"}
sox = {version = "*", markers = "platform_system == 'Linux' or platform_system == 'Darwin'"}
```

---

Ahh I couldn't find what macOS would've been called system wise, shoulda asked but yea thanks wasn't took tricky.